source/particle/output/interface.cc

```diff
@@ -44,23 +46,68 @@ namespace aspect
       template <int dim>
       void
       Interface<dim>::initialize ()
-      {}
+      {
+        file_index = 0;
```

Comment:
move this into the member initializer list of the constructor. `initialize()` would then just be an empty function